src/main/java/com/amihaiemil/docker/RtDocker.java

```diff
@@ -105,9 +105,10 @@ public final Images images() {
 
     @Override
     public final Networks networks() {
-        throw new UnsupportedOperationException(
-            "Networks API is not yet implemented. If you can contribute please,"
-            + " do it here: https://www.github.com/amihaiemil/docker-java-api"
+        return new ListedNetworks(
+                this.client,
+                URI.create(this.baseUri.toString() + "/networks"),
+                this
```

Comment:
@lumimul Same here, 4 spaces to the left please :)

Reply:
done